tests/unit/domain/test_metadata.py

```diff
@@ -164,3 +170,51 @@ def test_find_all_datastore_versions_when_draft_version_empty(mocker):
     actual = metadata.find_all_datastore_versions()
     assert len(actual['versions']) == 1
     assert actual['versions'][0]['version'] == '1.0.0.0'
+
+def test_find_all_metadata_skip_code_list_and_missing_values_for_data_structures(mocker):
+    
+    with open(DATA_STRUCTURES_FILE_PATH, encoding='utf-8') as f:
+        mocked_data_structures = json.load(f)
+
+    mocker.patch.object(
+        datastore, 'get_metadata_all',
+        return_value=mocked_data_structures
+    )
+    
+    filtered_metadata = metadata.find_all_metadata_skip_code_list_and_missing_values(version='1.0.0.0')
```

Comment:
I would prefer an expected metadata_all_no_code_list__1_0_0_0.json file to compare to. Easier to see the expected output this way imho.

Reply:
Not sure what you're asking here, please clarify

Reply:
I suppose we should have a positive test that checks for the removal of these lists and perhaps another that checks that we don't botch a file that already a-ok ?

Reply:
Implemented